src/compute/datomic_client_memdb/core.clj

```diff
@@ -61,11 +70,16 @@
 
   client-impl/Queryable
   (q [_ arg-map]
-    (apply peer/q (:query arg-map) (map (fn [x]
-                                          (if (instance? LocalDb x)
-                                            (.-db x)
-                                            x))
-                                        (:args arg-map))))
+    (let [{:keys [query args]} arg-map]
+      (when (collection-query? query)
+        (throw (ex-info "Only find-rel elements are allowed in client find-spec, see http://docs.datomic.com/query.html#grammar"
+                        {:cognitect.anomalies/category :cognitect.anomalies/incorrect,
+                         :cognitect.anomalies/message  "Only find-rel elements are allowed in client find-spec, see http://docs.datomic.com/query.html#grammar"})))
```

Comment:
When throwing the exception, the `:dbs` key is also included. Since it's easy, let's include it here too. Here's an example 

```clojure
clojure.lang.ExceptionInfo: Only find-rel elements are allowed in client find-spec, see https://docs.datomic.com/cloud/query/query-data-reference.html#query-grammar
(ex-data *e)
=>
{:cognitect.anomalies/category :cognitect.anomalies/incorrect,
 :cognitect.anomalies/message "Only find-rel elements are allowed in client find-spec, see https://docs.datomic.com/cloud/query/query-data-reference.html#query-grammar",
 :dbs [{:database-id "a7fad41d-0a86-4769-bfad-999391b35a43", :t 14828, :next-t 14829, :history false}]}
```

I think it'd be something like `:dbs (filterv #(instance? LocalDb %) args)`.

Reply:
That gives me something like `:dbs [#object[compute.datomic_client_memdb.core.LocalDb 0x316987c4 "compute.datomic_client_memdb.core.LocalDb@316987c4"]]`, which doesn't seem all that helpful.

It's easy to get a hold of  `:t` and `:next-t`, but I don't know where to look for `:database-id` and `:history`.

(Sidenote: TIL that I can't use `select-keys` in this case, because it uses `find` which isn't defined in terms of `clojure.lang.ILookup`.)

Reply:
> That gives me something like :dbs [#object[compute.datomic_client_memdb.core.LocalDb 0x316987c4 "compute.datomic_client_memdb.core.LocalDb@316987c4"]], which doesn't seem all that helpful.

It's a regular db, just like Datomic's. It implements ILookup [here](https://github.com/ComputeSoftware/datomic-client-memdb/blob/664a50ce15021521ac50d10563e39df94d8cf538/src/compute/datomic_client_memdb/core.clj#L70) for :t, :next-t. Our db doesn't have the nice printing like Datomic's has. That could be implemented but it's not a priority. PRs accepted :smiley: